scripts/services/getConfig.ts

```diff
@@ -595,13 +595,23 @@ export const getConfig: T.ExpectedExports.getConfig = async (effects) => {
           nullable: true,
           name: "Database Cache",
           description:
-            "How much RAM to allocate for caching the TXO set. Higher values improve syncing performance, but increase your chance of using up all your system's memory or corrupting your database in the event of an ungraceful shutdown. Set this high but comfortably below your system's total RAM during IBD, then turn down to 450 (or leave blank) once the sync completes.",
-          warning:
-            "WARNING: Increasing this value results in a higher chance of ungraceful shutdowns, which can leave your node unusable if it happens during the initial block download. Use this setting with caution. Be sure to set this back to the default (450 or leave blank) once your node is synced. DO NOT press the STOP button if your dbcache is large. Instead, set this number back to the default, hit save, and wait for bitcoind to restart on its own.",
+            "How much RAM to allocate for caching the TXO set. Higher values improve syncing performance, but may result in some re-work in the event of an ungraceful shutdown. 4-7GB is high enough to get most of the peformance benefit during IBD. Consider reducing this setting for lower resource devices (or a device with less available RAM)",
           range: "(0,*)",
+          default: 5_000,
           integral: true,
```

Comment:
For some reason GitHub doesn't let me push to your branch, so just need to remove the default of `dbcache`.

```suggestion
          range: "(0,*)",
          integral: true,
```